conf/app.routes

```diff
@@ -27,6 +27,7 @@ GET         /docs/api/preview                                       @uk.gov.hmrc
 GET         /docs/api/testdata/:service/:version                    @uk.gov.hmrc.apidocumentation.controllers.DocumentationController.fetchTestEndpointJson(service: String, version: String)
 GET         /docs/api/xml/:name                                     @uk.gov.hmrc.apidocumentation.controllers.DocumentationController.renderXmlApiDocumentation(name: String)
 GET         /docs/api/:service/:version/:endpoint                   @uk.gov.hmrc.apidocumentation.controllers.RedirectController.redirectToApiDocumentationPage(service: String, version: String, endpoint: String)
+GET         /docs/api/mtd-income-tax-service-guide                  @uk.gov.hmrc.apidocumentation.controllers.DocumentationController.mtdIncomeTaxServiceGuidePage()
```

Comment:
Should probably put "service-guide(s?)" in the url so that this can be duplicated for other future service guides using the same pattern.

Reply:
This is an interim tactical solution for the MTD Income Tax Service Guide as a "normal" page, just until a "proper" service guide microservice has been created, so it's not intended to set a precedent or pattern - don't think it needs changing.

Reply:
ok, we''l leave as is. We've done the other things that were commented on fyi.

Reply:
This is a one-off interim page for the MTD Income Tax Service Guide until we have a proper service guide microservice for it. So we're not looking to set a pattern or precedent.